modules/persistent-storage-csi-efs-troubleshooting.adoc

```diff
@@ -5,13 +5,13 @@
 // * storage/container_storage_interface/osd-persistent-storage-aws-efs-csi.adoc
 
 [id="efs-troubleshooting_{context}"]
-= AWS EFS troubleshooting
+= Amazon EFS troubleshooting
 
-The following information provides guidance on how to troubleshoot issues with AWS EFS:
+The following information provides guidance on how to troubleshoot issues with Amazon EFS:
 
-* The AWS EFS Operator and CSI driver run in namespace `openshift-cluster-csi-drivers`.
+* The Amazon EFS Operator and CSI driver run in namespace `openshift-cluster-csi-drivers`.
 
-* To initiate gathering of logs of the AWS EFS Operator and CSI driver, run the following command:
+* To initiate gathering of logs of the Amazon EFS Operator and CSI driver, run the following command:
```

Comment:
Revoked.

Reply:
@smunje1 It doesn't seem like this title is linking out to OCP resources or code. "Amazon EFS CSI driver" is the appropriate name to be used for the driver. Is there a reason I'm missing why this update should be revoked?

https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html

Reply:
Revoked. As the service name is associated with operator.